subsys/bluetooth/services/Kconfig.dis

```diff
@@ -117,6 +117,8 @@ config BT_DIS_FW_REV
 config BT_DIS_FW_REV_STR
 	string "Firmware revision"
 	depends on BT_DIS_FW_REV
+	default "$(APP_VERSION_TWEAK_STRING)" if "$(VERSION_MAJOR)" != ""
+	default "0.0.0+0"
```

Comment:
I think it makes sense to add to the commit message why `APP_VERSION_TWEAK_STRING` is used rather than `APP_VERSION_STRING` or `APP_VERSION_EXTENDED_STRING`. 

Does DIS set any requirements for the version?

Reply:
> Does DIS set any requirements for the version?

No, you can use whatever string (or format) you like. From the DIS specification:

_3.5 Firmware Revision String
The Firmware Revision String characteristic shall represent the firmware revision for the
firmware within the device._

Reply:
> I think it makes sense to add to the commit message why APP_VERSION_TWEAK_STRING is used rather than APP_VERSION_STRING or APP_VERSION_EXTENDED_STRING.

Sure, I will update it